dags/bqetl_artifact_deployment.py

```diff
@@ -124,11 +127,13 @@ def check_for_queued_runs(dag_id: str) -> bool:
             "script/bqetl view publish --add-managed-label --skip-authorized --project-id=moz-fx-data-marketing-prod && "
             "script/bqetl view publish --add-managed-label --skip-authorized --project-id=moz-fx-data-glam-prod-fca7 && "
             "script/bqetl view publish --add-managed-label --skip-authorized --project-id=moz-fx-data-shared-prod --target-project=mozdata --user-facing-only && "
+            "script/bqetl view publish --add-managed-label --skip-authorized --project-id=moz-fx-data-bq-people && "
             "script/bqetl view clean --skip-authorized --target-project=moz-fx-data-shared-prod && "
             "script/bqetl view clean --skip-authorized --target-project=moz-fx-data-experiments --project-id=moz-fx-data-experiments && "
             "script/bqetl view clean --skip-authorized --target-project=moz-fx-data-marketing-prod --project-id=moz-fx-data-marketing-prod && "
             "script/bqetl view clean --skip-authorized --target-project=moz-fx-data-glam-prod-fca7 --project-id=moz-fx-data-glam-prod-fca7 && "
             "script/bqetl view clean --skip-authorized --target-project=mozdata --user-facing-only && "
+            "script/bqetl view clean --skip-authorized --target-project=moz-fx-data-bq-people --project-id=moz-fx-data-bq-people && "
```

Comment:
`bqetl view clean` [trawls through all the non-derived datasets in the project looking for managed views](https://github.com/mozilla/bigquery-etl/blob/e958f73e26f1a62740bbb1ee5e925b75236990c4/bigquery_etl/cli/view.py#L339-L362), but permissions for Airflow GKE jobs to do that in the `moz-fx-data-bq-people` project don't appear to be [set up in cloudops-infra](https://github.com/mozilla-services/cloudops-infra/blob/master/projects/data-people/tf/modules/resources/main.tf) currently.

IMO granting the Airflow GKE service account `roles/bigquery.metadataViewer` on the `moz-fx-data-bq-people` project would be appropriate for this, as that would allow listing all datasets/tables/views and their associated metadata (e.g. labels) without allowing reading any actual data.

@whd what do you think?

Reply:
Or should we perhaps not do this at all, and instead insist that new HR ETLs go into datasets in `moz-fx-data-shared-prod`?

We did just move away from having derived ETLs/views in the `moz-fx-data-marketing-prod` project, so maybe we should do the same for `moz-fx-data-bq-people` (though it's possible there could be reasons why HR would want the data in `moz-fx-data-bq-people`).

Reply:
> though it's possible there could be reasons why HR would want the data in `moz-fx-data-bq-people`

@wwyc just reminded me that HR currently uses a separate Looker instance (though there are plans to consolidate our Looker instances), which is likely a reason to continue keeping HR data in `moz-fx-data-bq-people` for the time being.

Reply:
After paging in a bit of context via discussion with @wwyc, I'm of the opinion that moving HR ETLs to shared-prod/requiring new ones to be defined there is the appropriate strategy here. However, doing so is going to require a bit of policy discussion / decision making from the owners of `workgroup:people` first. Basically I'm going to get the ball rolling on extending https://mozilla-hub.atlassian.net/browse/DSRE-1437?focusedCommentId=793760 to allow for people data to be stored in the main data platform projects with the intent that we deprecate the separate people project entirely.

In the very short term, I think `roles/bigquery.metadataViewer` would be acceptable for airflow to have on the people project if you want to proceed with this as-is.

Reply:
> Basically I'm going to get the ball rolling on extending https://mozilla-hub.atlassian.net/browse/DSRE-1437?focusedCommentId=793760 to allow for people data to be stored in the main data platform projects with the intent that we deprecate the separate people project entirely.

Sounds good, though I think it could make sense for `moz-fx-data-bq-people` to continue to be used for HR's Fivetran syncs.

> In the very short term, I think `roles/bigquery.metadataViewer` would be acceptable for airflow to have on the people project if you want to proceed with this as-is.

I'd like to proceed with that, as I'm trying to get mozilla/private-bigquery-etl#443 deployed this week so the monthly snapshots can start running on September 1.

Would you like me to submit a `cloudops-infra` PR for granting Airflow that role on the `moz-fx-data-bq-people` project?

Reply:
> Sounds good, though I think it could make sense for moz-fx-data-bq-people to continue to be used for HR's Fivetran syncs.

I just got done writing https://mozilla-hub.atlassian.net/browse/DSRE-1742 where I opined that fivetran people data should probably be handled in the standard fivetran project. This is a representation I've made without actually knowing what data is specifically being synced this way: it was instead based on my knowledge of how the current fivetran project is administered.

> Would you like me to submit a `cloudops-infra` PR for granting Airflow that role on the `moz-fx-data-bq-people` project?

I can handle this tomorrow but if you'd like to file a PR that would be great as well

Reply:
> if you'd like to file a PR that would be great as well

mozilla-services/cloudops-infra/pull/5873